tests/unit/test_doc2row.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from langchain_core.documents import Document
+
+from langchain_google_cloud_sql_mysql.mysql_loader import (
+    DEFAULT_CONTENT_COL,
+    DEFAULT_METADATA_COL,
+    _parse_doc_from_row,
+    _parse_row_from_doc,
+)
+
+test_doc = Document(
+    page_content="Granny Smith 150 0.99",
+    metadata={"fruit-id": 1, "fruit_name": "Apple", "organic": 1},
+)
+cols_default = [DEFAULT_CONTENT_COL, DEFAULT_METADATA_COL]
+row_default = {
+    "page_content": "Granny Smith 150 0.99",
+    "langchain_metadata": {"fruit-id": 1, "fruit_name": "Apple", "organic": 1},
+}
+row_customized_flat = {
+    "fruit-id": 1,
+    "fruit_name": "Apple",
+    "variety": "Granny Smith",
+    "quantity_in_stock": 150,
+    "price_per_unit": 0.99,
+    "organic": 1,
+}
+row_customized_nested = {
+    "variety": "Granny Smith",
+    "quantity_in_stock": 150,
+    "price_per_unit": 0.99,
+    "langchain_metadata": {"fruit-id": 1, "fruit_name": "Apple", "organic": 1},
+}
+
+
+def test_row2doc_default():
+    assert (
+        _parse_doc_from_row([DEFAULT_CONTENT_COL], [DEFAULT_METADATA_COL], row_default)
+        == test_doc
+    )
```

Comment:
nit: if we have to use () to spread a line across multiple lines it probably doesn't need to be a single line in the first place. Please update the other tests as well.
```suggestion
    doc = _parse_doc_from_row(
        [DEFAULT_CONTENT_COL], [DEFAULT_METADATA_COL], row_default
    )
    assert doc == test_doc
```

Reply:
Sure, updated.